core/tests/vm-benchmark/src/instruction_counts.rs

```diff
@@ -0,0 +1,16 @@
+use vm_benchmark_harness::{cut_to_allowed_bytecode_size, get_deploy_tx, BenchmarkingVm};
```

Comment:
Nit: Doc-comments are always appreciated, e.g. a comment explaining the purpose of the binary as a topmost module doc-comment, and doc-comments for the main structures introduced (like `InstructionCounter`) help a lot.